Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -92,11 +130,22 @@ private void Initialize()
                 //Enable Boundary (example initializer)
                 AddManager(typeof(IMixedRealityBoundarySystem), new InputSystem.MixedRealityBoundaryManager());
             }
+
+            //TODO should this be optional?
+            //Sort the managers based on Priority
+            var orderedManagers = ActiveProfile.ActiveManagers.OrderBy(m => m.Value.Priority);
```

Comment:
I do like this, but let's try and order without Linq

Reply:
Lol, without linq, it's horrible for loops :P

Reply:
fine with me. *brute force*

Reply:
Can we keep this one Linq thing in, as there are no other "nice" solutions.  Plus it's is only used once on startup.

Reply:
Fine with me.